src/auxiliary/auxiliary.jl

```diff
@@ -146,6 +146,16 @@ given in the Trixi documentation regarding unstructured meshes.
 default_example_unstructured() = joinpath(examples_dir(), "unstructured_2d_dgsem", "elixir_euler_basic.jl")
 
 
+"""
+    ode_default_options()
+
+Return the default options for OrdinaryDiffEq's `solve`. Pass `ode_default_options()...` to `solve`
+to only return the solution at the final time and enable **MPI aware** error-based step size control.
+For example, use `solve(ode, alg; ode_default_options()...)`
+"""
+ode_default_options() = (; save_everystep = false, internalnorm = ode_norm, unstable_check = ode_unstable_check)
```

Comment:
I am wondering: Should we always use our internal norms? Or should we only use them where appropriate, i.e., in the parallel MPI case, and only expand to `(; save_everystep = false)` otherwise?

I am a bit wary of changing the default for all cases, since then we might get error reports from users who do not use this flag and have error messages that are slightly different to ours. But maybe I'm overly cautious here... 🤷‍♂️

Reply:
You mean something like

```julia
function ode_default_options()
  if mpi_isparallel()
    return (; save_everystep = false, internalnorm = ode_norm, unstable_check = ode_unstable_check)
  else
    return (; save_everystep = false)
  end
end
```
? Sounds reasonable to me since `ode_norm` is designed for MPI. So it would be a bit strange to also use it without MPI. What do you think @ranocha?

Reply:
Sounds reasonable

Reply:
Done in 495bcb53fefb8599805979707aad97f771cc7ba7.